### PR TITLE
Dev vcf 40 kb cred syscheck

### PR DESCRIPTION
**Pull request by kannanbalu**
Fixed Spelling mistake - ResponseException in 3 functions 1)Get-VCFCredentialTask 2) Get-VCFCredentialTask 3) Retry-VCFCredentialTask

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -2493,6 +2493,100 @@ Export-ModuleMember -Function Set-VCFDepotCredentials
...
 
+######### Start System Health Check ##########
+Function Start-PreCheckVCFSystem {
+
+<#
+    .SYNOPSIS
+    Performs system level health checks and upgrade pre-checks for an upgrade to be successful
+
+    .DESCRIPTION
+    Performs system level health checks and upgrade pre-checks for an upgrade to be successful
```
Comment (GaryJBlake) [inline powerVCF.psm1]: The Start-PreCheckVCFSystem cmdlet performs ...

File: powerVCF.psm1 [hunk 2]
```diff
@@ -2493,6 +2493,100 @@ Export-ModuleMember -Function Set-VCFDepotCredentials
...
+######### Start System Health Check Task Monitoring ##########
+
+Function Get-PreCheckVCFSystemTask {
+
+<#
+    .SYNOPSIS
+    Performs retrieval of a system precheck task that can be polled and monitored.
+
+    .DESCRIPTION
+    Performs retrieval of a system precheck task that can be polled and monitored.
```
Comment (GaryJBlake) [inline powerVCF.psm1]: The Get-PreCheckVCFSystemTask cmdlet perfroms ...

File: powerVCF.psm1 (hunk 1 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 2 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Fixed

**Comment by kannanbalu** [comment]
> You still need to create the Docs for each of these new modules. To do this you need to:
> 
> * Create a folder under docs > functions (for these new ones suggest Pre-Check System)
> * Create a .md file named after the module
> * Put the content from the ps1 file into the .md file and format it (use another file as an example)

Thanks Gary. I will work on it next and will update back.

**Pull request merged**